File: Pixly-FE/src/app/core/models/AdminDTOs/DashboardOverview.ts
import { List } from "lodash";
import { DailyUploadedPhotos } from "./DailyUploadedPhotos";
export interface DashboardOverview {
    totalUsers: number;
    totalPhotos: number;
    pendingPhotos: number;
    totalLikes: number;
    totalDownload: number;
    lastFewDayInfo : DailyUploadedPhotos[] ;
}

File: Pixly-BE/Pixly.Models/AdminDTO/DashboardOverview.cs
namespace Pixly.Models.AdminDTO
{
    public class DashboardOverview
    {
        public int TotalUsers { get; set; }
        public int TotalPhotos { get; set; }
        public int TotalLikes { get; set; }
        public int TotalDownload { get; set; }
        public int PendingPhotos { get; set; }
        public List<DailyUploadedPhotos> LastFewDayInfo { get; set; } = new();
    }
}
